src/App.js

```diff
@@ -3,14 +3,21 @@ import AddTodoForm from "./AddTodoForm";
 import TodoList from "./TodoList";
 
 function App() {
-  const [newTodo, setNewTodo] = useState('');
+  
+  const [todoList, setTodoList] = useState(['']);
```

Comment:
There's no need to have an empty string here `['']`.  You can just have an empty array `[]`